server_variable.js

```diff
@@ -204,7 +204,11 @@ function getAssessmentFromEnterpriseAPI(eventData, recaptcha) {
         expectedAction: recaptcha.action,
         userIpAddress: eventData.ip_override,
         userAgent: eventData.user_agent
-      }
+      },
+      assessmentEnvironment: {
```

Comment:
I would probably make both of these values a constant under //globals at the top. 

So, perhaps something like this:
```
const assessmentEnvironmentClient = 'github.com/GoogleCloudPlatform/recaptcha-enterprise-google-tag-manager';
const assessmentEnvironmentVersion = '1.0.0';
```

This will make it easier to find and adjust the version as necessary later since we don't yet have any kind of build process that would alter the files directly according to the version.